assets/css/admin.css

```diff
@@ -10,3 +10,54 @@
 	height: 32px;
 	width: 20px;
 }
+
+/* Environment Indicator Styling */
+#wpadminbar ul li#wp-admin-bar-tenup-experience-environment-indicator {
+	color: #fff;
+	pointer-events: none;
+}
+
+#wpadminbar ul li#wp-admin-bar-tenup-experience-environment-indicator.tenup-production {
```

Comment:
@psorensen I wonder if we could use less specificity. I can foresee client asking for different colors for production.

Reply:
Thanks for the feedback @rickalee - any suggestions here? Asking from a point of FEE ignorance 😃